packages/web/components/Badge/Badge.tsx

```diff
@@ -38,26 +181,23 @@ const getBadgeDescription = (badgeType: BadgeType): string =>
   `${getBadgeCopySubpath(badgeType)}.description`
 
 const BadgeComponent: React.FC<Props> = ({ badge }) => {
-  const { t } = useTranslation('common')
+  const { t } = useTranslation('badge')
 
   return (
     <>
       <div className="badge" title={t(getBadgeDescription(badge.type))}>
-        <div className="badge-icon" />
-        <span className="title">{t(getBadgeTitle(badge.type))}</span>
+        {/* <div className="badge-icon" />
+        <span className="title">{t(getBadgeTitle(badge.type))}</span> */}
+        <img src={`/images/badges/${badge.type}.svg`} alt="" />
```

Comment:
Can we use the badge title as the alt text? Does that interact with the `title` attribute on the div strangely? I think providing _some_ kind of alt test here would be nice for a11y purposes.

Reply:
Ah yes, I meant to come back and do this, what do you think? I made it so that the alt text should be "<badgeTitle>: <badgeDescription>"